pkg/query-service/app/traces/v3/query_builder.go

```diff
@@ -297,12 +303,27 @@ func buildTracesQuery(start, end, step int64, mq *v3.BuilderQuery, tableName str
 		query := fmt.Sprintf(queryTmpl, step, op, filterSubQuery, groupBy, having, orderBy)
 		return query, nil
 	case v3.AggregateOperatorNoOp:
-		// queryTmpl := constants.TracesSQLSelect + "from " + constants.SIGNOZ_TRACE_DBNAME + "." + constants.SIGNOZ_SPAN_INDEX_TABLENAME + " where %s %s"
-		// query := fmt.Sprintf(queryTmpl, spanIndexTableTimeFilter, filterSubQuery)
-		// return query, nil
-		return "", fmt.Errorf("not implemented, part of traces page")
+		var query string
+		if panelType == v3.PanelTypeTrace {
+			withSubQuery := fmt.Sprintf(constants.TracesExplorerViewSQLSelectWithSubQuery, spanIndexTableTimeFilter, filterSubQuery)
```

Comment:
I don't see anywhere `parentSpanID = ''`, which should always be added in the backend regardless of the user filters. We only should consider root span duration for the trace view.

Reply:
Added it